File: fastapi/models/delivery_city.go
package models

type DeliveryCity struct {
	ID          uint   `gorm:"primaryKey" json:"id"`
	Title       string `json:"title"`
	Door        int    `json:"door"`
	DoorDays    string `json:"door_days"`
	Storage     int    `json:"storage"`
	StorageDays string `json:"storage_days"`
	Fast        bool   `json:"fast"`
}

File: fastapi-rs/src/app/delivery_cities/entities/delivery_city.rs
use sea_orm::entity::prelude::*;
use serde::Serialize;

#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Serialize)]
#[sea_orm(table_name = "delivery_cities")]
pub struct Model {
    #[sea_orm(primary_key)]
    pub id: i64,
    pub title: String,
    pub door: i32,
    pub door_days: String,
    pub storage: Option<i32>,
    pub storage_days: Option<String>,
    pub fast: bool,
}

#[derive(Copy, Clone, Debug, EnumIter)]
pub enum Relation {}

impl RelationTrait for Relation {
    fn def(&self) -> RelationDef {
        panic!("No RelationDef")
    }
}

impl ActiveModelBehavior for ActiveModel {}
